R/tax_range_strat.R

```diff
@@ -150,26 +157,32 @@ tax_range_strat <- function(occdf, name = "genus", level = "bed",
 
   #Create object to hold information
   if (is.null(certainty)) {
-    ranges <- data.frame(taxon = unique_taxa, min_bin = NA, max_bin = NA)
+    ranges <- data.frame(taxon = unique_taxa, group = NA, min_bin = NA,
```

Comment:
Should we add some cleanup at the end to remove this column if group = NULL? Otherwise we're just getting a column of NAs.